__tests__/workers/postAddedSlackChannelSend.ts

```diff
@@ -118,6 +161,49 @@ describe('postAddedSlackChannelSend worker', () => {
     });
   });
 
+  it('should send a message to the slack channel for squad', async () => {
+    await con.getRepository(Post).save([
+      {
+        ...postsFixture[0],
+        id: 'squadslackchannelp1',
+        title: 'Squad Channel Post 1',
+        shortId: 'sschp1',
+        url: 'http://localhost:5002/posts/squadslackchannelp1',
+        canonicalUrl: 'http://localhost:5002/posts/squadslackchannelp1',
+        sourceId: 'squadslackchannel',
+      },
+    ]);
+    const post = await con.getRepository(ArticlePost).findOneByOrFail({
+      id: 'squadslackchannelp1',
+    });
+
+    await expectSuccessfulTypedBackground(worker, {
+      post: post as unknown as ChangeObject<ArticlePost>,
+    });
+
+    expect(conversationsJoin).toHaveBeenCalledTimes(1);
+    expect(chatPostMessage).toHaveBeenCalledTimes(1);
+
+    expect(conversationsJoin).toHaveBeenCalledWith({
+      channel: '1',
+    });
+    expect(chatPostMessage).toHaveBeenCalledWith({
+      channel: '1',
+      attachments: [
+        {
+          author_icon: 'https://app.daily.dev/apple-touch-icon.png',
+          author_name: 'daily.dev',
+          image_url: 'https://daily.dev/image.jpg',
+          title: 'Squad Channel Post 1',
+          title_link:
```

Comment:
Do you maybe also want to test public squads for the diff or we already do so?

Reply:
When I add the the private check in the worker the response handling will be the same as for source so no need I think. https://github.com/dailydotdev/daily-api/pull/2113#discussion_r1709237010

Reply:
Solved in bd11f05533a7bd445c602bd8c69e12344ea8d816